src/main/java/com/encora/Tree.java

```diff
@@ -10,6 +10,7 @@
     List<Integer> inOrder();
     List<Integer> preOrder();
     List<Integer> postOrder();
-    String toStringOpt(String opt);
     int getRoot();
+    Order getOrder();
```

Comment:
For an interface, meaning a contract, I don't think adding getter/setter method is a good idea.